schemas/mw-revision-visibilityset.yaml

```diff
@@ -0,0 +1,62 @@
+title: MediaWiki Revision Visibility Set
```

Comment:
Docs around the log event call it "Change visibility".  See https://www.mediawiki.org/wiki/Manual:Log_actions.